src/Renci.SshNet/Messages/Message.cs

```diff
@@ -37,7 +37,7 @@ protected override void WriteBytes(SshDataStream stream)
             base.WriteBytes(stream);
         }
 
-        internal byte[] GetPacket(byte paddingMultiplier, Compressor compressor)
```

Comment:
fyi @jacobslusser (or anyone else interested), the Message->Socket logic (in `Session.SendMessage` and `Message.GetPacket`) is somewhat cumbersome. There is probably some efficiency to be gained in untangling it.